src/IssueViz.UnitTests/Editor/DocumentNavigatorTests.cs

```diff
@@ -64,10 +64,15 @@ public void TestInitialize()
                 .Setup(x => x.GetEditorOperations(mockTextView))
                 .Returns(editorOperationsMock.Object);
 
+            threadHandlingMock = new Mock<IThreadHandling>();
+            threadHandlingMock.Setup(x => x.RunOnUIThread(It.IsAny<Action>())).Callback<Action>(op => op());
```

Comment:
This is not great however, I think its the best I can do with the way this testclass is written.